Docker build doesn't run: The 'util' package was not installed in a way that PackageLoader understands.

**Describe the bug**

After building the docker image myself, it fails to run.

**To Reproduce**

1. Clone latest git master
2. Run `docker build .`
3. Run `docker run <image>`

Output:

```
[2021-06-12 09:20:31 +0000] [11] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/gunicorn/arbiter.py", line 583, in spawn_worker
    worker.init_process()
  File "/usr/lib/python3/dist-packages/uvicorn/workers.py", line 57, in init_process
    super(UvicornWorker, self).init_process()
  File "/usr/lib/python3/dist-packages/gunicorn/workers/base.py", line 119, in init_process
    self.load_wsgi()
  File "/usr/lib/python3/dist-packages/gunicorn/workers/base.py", line 144, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/usr/lib/python3/dist-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/usr/lib/python3/dist-packages/gunicorn/app/wsgiapp.py", line 49, in load
    return self.load_wsgiapp()
  File "/usr/lib/python3/dist-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/usr/lib/python3/dist-packages/gunicorn/util.py", line 383, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/app/main.py", line 10, in <module>
    import db.wireguard
  File "/app/db/wireguard.py", line 10, in <module>
    import script.wireguard
  File "/app/script/wireguard.py", line 17, in <module>
    import util
  File "/app/util.py", line 2, in <module>
    jinja_env = Environment(loader=PackageLoader(__name__, 'templates'))
  File "/usr/local/lib/python3.8/dist-packages/jinja2/loaders.py", line 309, in __init__
    raise ValueError(
ValueError: The 'util' package was not installed in a way that PackageLoader understands.
```

**Platform:**
 - OS: Arch Linux with docker 20.10.7
 - Version: git master 0aec91c6faed97f809981131d31f54f286d34b9e

--- a/wg_dashboard_backend/util.py
+++ b/wg_dashboard_backend/util.py
@@ -1,2 +1,4 @@
-from jinja2 import Environment, PackageLoader
-jinja_env = Environment(loader=PackageLoader(__name__, 'templates'))
+import os
+from jinja2 import Environment, FileSystemLoader
+templates_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'templates')
+jinja_env = Environment(loader=FileSystemLoader(templates_path))

